MIPS/mip-0005-mip-upgrade.md

```diff
@@ -0,0 +1,315 @@
+The Mina Foundation will create a pull request to update the Readme file in the MIPs section in GitHub.
+https://docs.google.com/document/d/1ywIopypVE7tRLAD3909Tu5jjxxC1qmH7amUj2GJPHNM/edit
+
+**Add new pages in GitHub for both MIP Facilitators and MIP Reviewers:** 
+The Mina Foundation will create a pull request to create the new pages for these MIP roles inside the MIPs section in GitHub. 
+https://docs.google.com/document/d/1HPHJ77VCbvUL0nG8jC4o2B8D_ly-r4tOPGuOwAqx9Rs/edit
+https://docs.google.com/document/d/1PouLedjwCV0m3Qi-Kfe-oibkNmJMRQCjXWWSF2Yk2v8/edit
+
+**Add a new page in GitHub called MIP template:**
+The Mina Foundation will create a pull request to create a new page to host the MIP template inside the MIPs section in GitHub.
+https://docs.google.com/document/d/1WUORti_GyXikuGPIf9Xxr7otauDzidDhk__ElDVAfC0/edit
+
+**Add new communication channels in Discord for MIP Facilitators and MIP Reviewers:**
+The Admins of the Mina Discord Server will create new private channels to coordinate the operations and internal discussion for both of these roles, granting MIP Facilitators the necessary permissions to effectively facilitate communications and moderate these channels as needed. 
+
+**Implementation plan**
+The Mina Foundation, in collaboration with o1 Labs and other members of the Mina ecosystem, will support the implementation of the proposed changes and help initiate this new process by temporarily filling the new roles as needed while the processes are tested, and the community develops the capacity to effectively assume these roles.
+
+These changes should be implemented, and the new roles filled by elected community members, no later than one year after approval through an on-chain vote. The Mina Foundation will provide regular updates and coordinate communications throughout the implementation phase.
+
+
+## Security Considerations
+
+**Dangerous proposals passing with low turnout**
+The proposal introduces Risk Assessments for informational purposes, allowing the community to review risks before voting. However, the proposal depends on the community to stay informed and reject dangerous proposals through on-chain voting. To further mitigate this, future upgrades could consider linking risk levels to higher participation thresholds for high-risk MIPs.
+
+**Concentrating too much power in the hands of a few experts**
+The risk of centralizing authority in the hands of a few reviewers is mitigated through transparency and broader community involvement. Public debates, deliberation tools, and well-defined reviewer roles ensure decisions reflect diverse perspectives and are not overly influenced by a small group.
+
+**Risk of not having enough experts**
+The proposal addresses the potential lack of expertise by assigning MIP Facilitators to manage the process, ensuring that a sufficient number of expert reviews are collected. If expertise is lacking, the MIP may be put on hold until necessary input is secured.
```

Comment:
Consider whether all Reviewers really need to be elected. May be helpful to allow for any community member to volunteer to review a proposal? Perhaps with consent from elected experts to ensure expertise?

Reply:
Perhaps we can at least have an initially proposed list of reviewers submitted inside of this MIP? And then the community can elect or remove individual members from then on out.

Reply:
Any community member is already welcome to review and provide feedback on any proposal, and this won’t change with the upgrade. The introduction of elected experts aims to ensure that sufficient time and expertise are dedicated to creating a comprehensive risk assessment for the community. I can work on making this distinction clearer in the proposal.